Cargo.toml

```diff
@@ -26,6 +26,14 @@ name = "lightningcss"
 path = "src/lib.rs"
 crate-type = ["rlib"]
 
+[features]
```

Comment:
Moved the feature flags up since they're commonly found above the dependencies section. 
When evaluating a crate I find it nice to be able to quickly see what features I can turn off.

Happy to move them back if you prefer!